lib/slice.ark

```diff
@@ -0,0 +1,16 @@
+{
+  (let slice (fun (string escapeCharacter){
+    #made by Natendrtfm <3
+        (mut i 0)
+        (mut word "")
+        (mut letter "")
+        (mut sliccedList [])
+        (while (< i (len string) ){
+            (set letter (@ string i))
+            (if (= letter escapeCharacter){(set sliccedList (append sliccedList word))(set word "")} ())
```

Comment:
better putting the if on multiple lines, for a better readability, like so:
```clojure
(if (= letter escapeChar)
    {
        ...
    }
    ())
```

Reply:
Updated, now it's multiple lines